lib/db.js

```diff
@@ -190,7 +190,6 @@ function net_db(params, callback) {
       logger.warning('Problem contacting DITCH server: ' + err.message + "\n" + util.inspect(err.stack));
       return callback(err);
     });
-    req.end();
```

Comment:
Looks like we lost all the git history of this component with open sourcing it. I could not figure out when and why this was added. Request objects should fire the `end` event if no more data is provided. Maybe instead of removing it you can try this:

```javascript
req.on('end', function () {
    callback(null, {stream: req});
});
```

(Return only when the stream has finished)